source/2020-01-10-the-ember-times-issue-130.md

```diff
@@ -0,0 +1,237 @@
+
+The new Octane syntax took a bit of getting used to, particularly around how to make use of tracked properties in place of computed properties, but the team is enjoying using the new auto-tracking system and class based components in many places throughout the application, and plans on incrementally updating all of their components over time.
+
+Almost all new work that gets done on the application is written with Glimmer components and in a clean and idiomatic Ember Octane way.
+
+<!-- alex ignore mental -->
+Some of the big benefits that the Outdoorsy team experienced from Octane are a simplified mental model of how Ember works, improved code readability, and easier reasoning about what context a state is derived from. Not having to reference several files to determine where a property is initially defined was definitely a boon to developer productivity.
+
+It’s great to hear that Octane is working really well in a production application at the scale and complexity of Outdoorsy’s.
+
+We know there are lots of other great examples of Octane being used in the wild. If you’d like to share your experiences, please feel to reach out to the Ember Times team.
+
+---
+
+## [How to Migrate to Octane 🐹🏗️](https://guides.emberjs.com/release/upgrading/current-edition/)
+
+<!--alex ignore nuts-->
+Ember Octane represents a big shift in Ember's syntax, features, and mental models. There is, however, no need for a stop-the-world migration, Octane provides an opt-in method of migration where you can apply each core feature enhancement incrementally.
+
+If you are migrating an existing Ember application to Octane we have an excellent list of resources to help you out:
+
+- [The Ember Guides: Octane Update Guide](https://guides.emberjs.com/release/upgrading/current-edition/)
+- [The Ember Octane vs. Classic Ember Cheat Sheet](https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet/)
+- [Ember Atlas Recommended Migration Order](https://www.notion.so/Atlas-Recommended-Migration-Order-a22f948a7cce4e01896d674f727bee74)
+
+Whatever you end up doing, you won’t have to do it alone as members of the community have generously contributed [numerous codemods](https://github.com/ember-codemods) to help us to migrate efficiently.
+
+- [Angle Brackets Codemod](https://github.com/ember-codemods/ember-angle-brackets-codemod)
+- [Tracked Properties Codemod](https://github.com/ember-codemods/ember-tracked-properties-codemod)
+- [Native Class Codemod](https://github.com/ember-codemods/ember-native-class-codemod)
+- [No Implicit This or {{foo}} to {{this.foo}} Codemod](https://github.com/ember-codemods/ember-no-implicit-this-codemod)
+- [ES5 Getter Setter Codemod](https://github.com/ember-codemods/es5-getter-ember-codemod)
```

Comment:
Nit: This should be just `ES5 Getter Codemod` since there is no setter functionality with this codemod

Reply:
@suchitadoshi1987 much appreciated! just committed a fix